lib/pancake.js

```diff
@@ -255,27 +255,28 @@ class Pancake {
       // if metadata is not defined, skip this build
       return;
     }
-    const properties = {
-      'createdAt': {
-        'type': 'Date',
-        'defaultFn': 'now',
-      },
-      'modifiedAt': {
-        'type': 'Date',
-        'defaultFn': 'now',
-      },
-    };
+    const properties = {};
```

Comment:
A user of this program may need or may not need createdAt and modifiedAt for ALL MODELS. If someone need them (or any other attributes he want to share between all his persisted models), he could declare it in a base class.